tests/ShapeFileTest.php

```diff
@@ -316,11 +316,12 @@ public function testShapeName(): void
      */
     public function testShapeSaveLoad(int $shapeType, array $points): void
     {
+        $shapeEnum = ShapeType::tryFrom($shapeType) ?? ShapeType::Unknown;
```

Comment:
You can pass a enum directly instead of an int.
```suggestion
    public function testShapeSaveLoad(ShapeType $shapeType, array $points): void
    {
```